src/test/java/org/mockitousage/verification/VerificationWithTimeoutTest.java

```diff
@@ -45,25 +45,25 @@
 
     private ScheduledExecutorService executor;
 
+    private DelayedExecution delayedExecution;
+
+
+    
+    
     @Before
     public void setUp() {
-        executor = newSingleThreadScheduledExecutor();
+        delayedExecution = new DelayedExecution();
```

Comment:
The before method seems overkill now, compared to a simple field initialisation.